modern/src/App.js

```diff
@@ -682,6 +683,7 @@ function Modern() {
       </DropTarget>
       <select
         style={{ position: "absolute", bottom: 0 }}
+        value={skinUrl}
```

Comment:
Could we just parse this out of `window.location`? Maybe we could have the current value live in a local component state value which is synced with the actual url via a useEffect hook?

Reply:
yea, lemme give that a try...kinda realized that after putting it in the store hah

Reply:
haha, that was a lot easier than I thought.